core/src/test/java/io/fabric8/maven/core/util/MavenUtilTest.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2016 Red Hat, Inc.
+ *
+ * Red Hat licenses this file to you under the Apache License, version
+ * 2.0 (the "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package io.fabric8.maven.core.util;
+
+import java.util.Properties;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+/**
+ * Checking the behaviour of utility methods.
+ */
+public class MavenUtilTest {
+
+
+    @Test
+    public void testYamlToPropertiesParsing() {
```

Comment:
Some more tests would be awesome, maybe with a dedicated test file (not reusing an exisiting one, as it makes tests dependant on each other).

Something like multiple arrays after each other, or null values, ...